docs/src/main/asciidoc/writing-native-applications-tips.adoc

```diff
@@ -70,6 +73,9 @@ Now, let's tell the `native-image` executable about this configuration file by t
 </plugin>
 ----
 
+When using `gradle` add extra arguments to your gradle command `--additional-build-args='-H:ResourceConfigurationFiles=resources/main/resources-config.json'`
```

Comment:
Is there a way to include it in the build file? Because adding it to the command line doesn't seem like a user friendly solution.

Reply:
Thanks for pointing this out, this is indeed also possible : 
`buildNative {
    additionalBuildArgs = [
            '-H:DynamicProxyConfigurationFiles=resources/main/resources-config.json'
    ]
}`
It makes it indeed better since the configuration can be in version control as well.